src/cocoa/toga_cocoa/app.py

```diff
@@ -123,7 +123,6 @@ def create(self):
         formal_name = self.interface.formal_name
 
         self.interface.commands.add(
-            toga.Command(None, 'About ' + formal_name, group=toga.Group.APP),
```

Comment:
I understand the intention here, but I don't think this will work. In an idiomatic macOS app, `About` appears in the app menu, not the help menu, and has the name "About MyApp".

I suspect that the menu *construction* will need to remain an impl-layer thing; although the actual commands can be made interface layer.